09-gis.Rmd

```diff
@@ -36,7 +36,7 @@ of flexibility and power waiting for you with the command line. Many times
 you can do something on the command line in a fraction of the time you
 can do it with a GUI.
 
-The 'CLI vs GUI' debate can be adversial but it does not have to be; both options can be used interchangeably, depending on the task at hand and the user's skillset.^[GRASS GIS and PostGIS are popular in academia and industry and can be seen as products which buck this trend as they are built around the command-line.
+The 'CLI vs GUI' debate can be adversial but it does not have to be: both options can be used interchangeably, depending on the task at hand and the user's skillset.^[GRASS GIS and PostGIS are popular in academia and industry and can be seen as products which buck this trend as they are built around the command-line.
```

Comment:
the copyediting suggested to change from colon to semicolon (see: https://github.com/Robinlovelace/geocompr/commit/b13212daebabf870943f0d57375aabf3318f2a24), so I would stick with the semicolon